docs/UserGuide.md

```diff
@@ -121,6 +121,13 @@ Check out the screenshot of each command for an idea of the expected output in t
 
 <div style="page-break-after: always;"></div>
 
+<div markdown="span" class="alert alert-warning">:bulb: **Tip:**
+Do not worry if you input any commands wrongly when using our app! Our app gives helpful error messages depending on the command you entered, 
+and you can always enter the [help command](#5-view-help) if you need to refer back to the various commands avaliable! 
+</div>
+
+<div style="page-break-after: always;"></div>
+
```

Comment:
Might want to consider moving this to the miscellaneous section. The commands section currently fits into exactly one page and this extra tip will awkwardly carry over to the next page.